### ISSUE TITLE
Cannot flash Android 11 GSI on my Asus Zenfone Go

### ISSUE THREAD

**Issue by Iuspeppe88** [phhusson/treble_experimentations#1796]
Hi! Until now I've flased Android 9 and 10 on my device and everything was fine during the flashing. But when I try to flash Android 11 GSI release via fastboot I get this error: FAILED (remote Unknown chunk type). System partition type of my device (Asus Zenfone Go) is A only. 

Thanks to those who will help me and sorry for my bad english!

**Comment by phhusson** [comment]
Which gsi variant do you use?

9 avr. 2021 19:04:57 Iuspeppe88 ***@***.***>:

**Comment by Iuspeppe88** [comment]
> 
> Which gsi variant do you use? 9 avr. 2021 19:04:57 Iuspeppe88 ***@***.***>:

I use "system-roar-arm-aonly-vanilla.img"

**Comment by Iuspeppe88** [comment]
It's normal that the filename extension is "img-1" instead of "img"? AOSP 10 images was "img" only, but these with Android 11 end with "img-1". I try to change extension to img, but I get the same error "FAILED (remote Unknown chunk type)". I don't know where I'm wrong :(

**Comment by youcancallmekhen** [comment]
> It's normal that the filename extension is "img-1" instead of "img"? AOSP 10 images was "img" only, but these with Android 11 end with "img-1". I try to change extension to img, but I get the same error "FAILED (remote Unknown chunk type)". I don't know where I'm wrong :(

> For some reason, it seems the img file you downloaded was split into multiple files, that's why the file extension becomes img-1. You need to download the file again.

**Comment by Iuspeppe88** [comment]
> For some reason, it seems the img file you downloaded was split into multiple files, that's why the file extension becomes img-1. You need to download the file again.

I downloaded the image several times and the extension is always "img-1". But, as I said before, this only happens when I try to download AOSP 11 GSI, no problem with AOSP 9 or 10.

**Comment by ShahrukhAbrar** [comment]
> > For some reason, it seems the img file you downloaded was split into multiple files, that's why the file extension becomes img-1. You need to download the file again.
> 
> I downloaded the image several times and the extension is always "img-1". But, as I said before, this only happens when I try to download AOSP 11 GSI, no problem with AOSP 9 or 10.

Can you send me the  url of the file? I can try downloading and check if it a universal prob.

**Comment by Iuspeppe88** [comment]
Newest release are back to "img" extension, but the issue remain :(
![Immagine](https://user-images.githubusercontent.com/82230630/121607951-409b1880-ca51-11eb-9891-b03fd954dcf2.png)

**Comment by phhusson** [comment]
That's going to sound weird, but you might need to use an older `fastboot` tool on your computer (the "chunk type" is created by `fastboot` computer-side when the file being flashed is not sparse)

**Comment by Iuspeppe88** [comment]
> That's going to sound weird, but you might need to use an older `fastboot` tool on your computer (the "chunk type" is created by `fastboot` computer-side when the file being flashed is not sparse)

It's weird 'cause I haven't that issue when I try to flash AOSP 9 or 10 with the same `fastboot` tool. However, could you suggest me an alternative `fastboot` tool that I can try? Thank you for your help ^^

**Comment by phhusson** [comment]
Yes my aosp 9 and aosp 10 releases are sparse, which means it's not sparsed by fastboot

11 juin 2021 12:07:38 Iuspeppe88 ***@***.***>:

**Comment by Iuspeppe88** [comment]
> Yes my aosp 9 and aosp 10 releases are sparse, which means it's not sparsed by fastboot 

Ok, thank you! I'll try with an older version of `fastboot` tool and I'll let you know ;)

**Comment by Iuspeppe88** [comment]
So, I tried different versions of the `fastboot` tool, but I alway get the same error "FAILED (remote Unknown chunk type)". I found this tool called "SparseConverter" which I've converted your AOSP 11 in a sparse img. When I try to flash the converted img it no more get me the message "Invalid sparse file format at header magic"... but it still ends with "FAILED (remote Unknown chunk type)" lol! I'm desperate XD
![Immagine](https://user-images.githubusercontent.com/82230630/121731554-68da5400-caf1-11eb-8c69-8b280897a2c3.png)

**Comment by JonnyTech** [comment]
Are you able to try to fastboot flash with Linux rather than Windows? (even from a live usb medium)

**Comment by Iuspeppe88** [comment]
> Are you able to try to fastboot flash with Linux rather than Windows? (even from a live usb medium)

Uhm yeah, I can. I'll try with a virtual machine and I'll let you know. Thanks ^^

**Comment by Iuspeppe88** [comment]
@JonnyTech same error with Linux :(
![Immagine](https://user-images.githubusercontent.com/82230630/121788265-394c4a00-cbcc-11eb-9fb7-7d61cfeaf9cf.png)

**Comment by JonnyTech** [comment]
Hmm, are you sure that you have unloaded your phone's bootloader? I get that message with some (re)locked bootloaders.

Please try these commands to correctly download and extract the image:

```
wget https://github.com/phhusson/treble_experimentations/releases/download/v309/system-roar-arm-aonly-vanilla.img.xz
xz -dk system-roar-arm-aonly-vanilla.img.xz
```

Then the following to verify it:
```
file system-roar-arm-aonly-vanilla.img
system-roar-arm-aonly-vanilla.img: Linux rev 1.0 ext2 filesystem data, UUID=8a90a354-6fbf-42bb-9f00-d1e1eeecf4b3 (extents) (large files) (huge files)

sha256sum system-roar-arm-aonly-vanilla.img
982f7dcd748f8cdfa319d861e049599112e815ec486e222c609cf921ac9f15bb  system-roar-arm-aonly-vanilla.img

sha1sum system-roar-arm-aonly-vanilla.img
409bb9ff9c848d045766bfd6e6d8a08fb80bd2c1  system-roar-arm-aonly-vanilla.img
```

If the checksum matches then flash it:
```
sudo flash system system-roar-arm-aonly-vanilla.img
```

Otherwise you may need to do a full stock update (all partitions) in order to get a working bootloader.

**Comment by Iuspeppe88** [comment]
> Hmm, are you sure that you have unlocked your phone's bootloader? I get that message with some (re)locked bootloaders.

Yes, 'cause flashing AOSP 9 and 10 GSI does not give me this error. I get that message only when I try to flash AOSP 11 GSI. 

> Then the following to verify it:
![Immagine](https://user-images.githubusercontent.com/82230630/121821518-d6c17f80-cc99-11eb-8249-d0a328c7163c.png)

> If the checksum matches then flash it:
![Immagine2](https://user-images.githubusercontent.com/82230630/121823163-ab439280-cca3-11eb-905f-c1508ec57b9e.png)

> Otherwise you may need to do a full stock update (all partitions) in order to get a working bootloader.

Hmm, I don’t think I can... I’m not that experienced ahah XD. Anyway, thank you ^^
